src/main/java/org/generationcp/middleware/manager/api/GermplasmDataManager.java

```diff
@@ -1191,4 +1192,11 @@
 	long countGermplasmByStudy(Integer studyDbId);
 
 	List<GermplasmDTO> getGermplasmByStudy(Integer studyDbId, Integer pageNumber, Integer pageSize);
+
+	List<AttributeDTO> getAttributesByGid(
```

Comment:
@vmaletta remove final from the parameters of the new functions in the interface

Reply:
ok